incubator/cockroachdb/values.yaml

```diff
@@ -0,0 +1,17 @@
+# Default values for cockroachdb.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Name: "cockroachdb"
+Image: "cockroachdb/cockroach"
+ImageTag: "latest" # Keep at latest until 1.0, then switch to a specific version
```

Comment:
Can we do this as a specific version?  `latest` has taken down production for me a couple of times :)  Not a fan of latest.